test/setup.jest.ts

```diff
@@ -40,4 +46,94 @@ jest.mock('@elastic/eui/lib/eui_components/icon', () => ({
   ICON_COLORS: require('@elastic/eui/lib/eui_components/icon/icon').COLORS,
 }));
 
+jest.mock('moment', () => {
+  const moment = jest.requireActual('moment');
+
+  // Set moment tz mock
+  if (!moment.tz) moment.tz = {};
+  moment.tz.names = () => ['Pacific/Tahiti'];
+  const momentInstance = moment();
+
+  momentInstance.format = jest.fn().mockReturnValue('2023-01-25T10:05');
+
+  function fakeMoment() {
+    return momentInstance;
+  }
+
+  Object.assign(fakeMoment, moment);
+
+  return fakeMoment;
+});
+
+const mockUseContext = {
+  notificationsService: {
+    getChannels: () => {
+      return {
+        ok: true,
+        response: {
+          channel_list: [],
+        },
+      };
+    },
+  },
+  indexService: {
+    getIndices: () => {
+      return {
+        ok: true,
+        response: {
+          indices: [],
+        },
+      };
+    },
+  },
+  detectorsService: {
+    getDetectors: () => {
+      return {
+        ok: true,
+        response: {
+          hits: {
+            hits: [mockDetectorHit],
+          },
+        },
+      };
+    },
+  },
+  ruleService: {
+    fetchRules: () => {
+      return Promise.resolve([mockDetectorHit]);
+    },
+    getRules: () => {
+      return {
+        ok: true,
+        response: {
+          hits: {
+            hits: [mockDetectorHit],
+          },
+        },
+      };
+    },
+  },
+};
+
+jest.mock('react', () => {
+  const ActualReact = jest.requireActual('react');
+  return {
+    ...ActualReact,
+    useContext: () => mockUseContext,
+  };
+});
+
+jest.mock('../public/pages/CreateDetector/components/ConfigureAlerts/utils/constants.ts', () => ({
+  ...jest.requireActual(
+    '../public/pages/CreateDetector/components/ConfigureAlerts/utils/constants.ts'
+  ),
+  ALERT_SEVERITY_OPTIONS: {
```

Comment:
Is this constant only used for testing? If not can it be shared with app code to avoid divergence later?